rules.txt

```diff
@@ -22,12 +22,12 @@
 5.  The Game
   5.1  Games shall be played best of 3 to 21.
     5.2  The winning team must win by 2 points.
-    5.3  The winning point must be the result of a lob.
-    5.4  The Fillipini Rule
-      5.4.1  If the team on game point scores a point that is not a lob the opponents shall incur a strike.
-      5.4.2  If the team incurring a strike shall score 1 point,  their strikes shall be reset.
-      5.4.3  If a team incurs 3 strikes they shall loose the game.
-      5.4.4  Members of all teams are required to acknowledge the Fillipini rule upon any team reaching game point by verbally confirming that "The Fillipini rule is in effect".   
+    5.3  The winning point must be the result of a lob, except in the case of 5.4, the Three-Strike (Filippini) Rule.
+    5.4  The Three-Strike (Filippini) Rule
+      5.4.1  If a team is within one point of a winning score and scores a point that is not the result of a lob, their opponents shall incur a strike.
+      5.4.2  If the opponent of a the team within one point of a winning score scores any number of points, their strikes shall be reset.
```

Comment:
How about simplifying this to something like `If a team earns at least one (1) point, their strikes shall be reset to zero (0)`.

I can't think of a scenario where that isn't true (and matters).